insights/parsers/installed_rpms.py

```diff
@@ -253,24 +253,21 @@ def __init__(self, *args, **kwargs):
 
     def parse_content(self, content):
         packages = defaultdict(list)
-        for line in get_active_lines(content, comment_char='COMMAND>'):
+        if content and "COMMAND>" in content[0]:
+            content = content[1:]
```

Comment:
better raise a "SkipComponent" when content is empty before and after this removal